cmd/iam.go

```diff
@@ -532,6 +533,16 @@ func (sys *IAMSys) DeletePolicy(ctx context.Context, policyName string, notifyPe
 		return errServerNotInitialized
 	}
 
+	for _, v := range iampolicy.DefaultPolicies {
+		if v.Name == policyName {
+			// Path would be same on all endpoint
+			policyDocPath := pathJoin(globalEndpoints[0].Endpoints[0].Path, minioMetaBucket, getPolicyDocPath(policyName))
+			if _, err := os.Stat(policyDocPath); err != nil {
+				return fmt.Errorf("inbuilt policy `%s` not allowed to be deleted", policyName)
```

Comment:
Though this may work, I believe we can be more consistent in the code and use 'loadPolicyDoc()' function instead

Reply:
Just implement a function called statConfig() - just like readConfig() so that you want to check for existence on the disk.